src/utils.ts

```diff
@@ -225,19 +233,58 @@ export const getAngle = (posX: number, posY: number) => {
   return (Math.atan2(posY, posX) * 180) / Math.PI;
 };
 
+export const isCssPropsFromAxes = (originalCssProps: {
+  [key: string]: string;
+}) => {
+  let same = true;
+  Object.keys(PREVENT_DRAG_CSSPROPS).forEach((prop) => {
+    if (
+      !originalCssProps ||
+      originalCssProps[prop] !== PREVENT_DRAG_CSSPROPS[prop]
+    ) {
+      same = false;
+    }
+  });
+  return same;
+};
```

Comment:
"touch-action" looks missing, shouldn't it have to be included?
You're overriding it in Axes in L269

Reply:
"touch-action" property can be any value from PanInput's touchAction option. 
Therefore, It is not included in the logic for determining whether it is from common CSS property from Axes.

Reply:
In that case, shouldn't there be any logic/property that determines that?

Reply:
How about adding a custom css property like `--egjs-axes-flag: true` to the element to identify it?
When custom css property is added, there is no need to check other properties when determining whether it has already been modified by Axes, the purpose of isCssPropsFromAxes.

Reply:
I think you can add an additional property in `originalCSSProps` whether it should be reverted if you have to.